src/styles/variables.scss

```diff
@@ -0,0 +1,3 @@
+$main-accent: #060b35;
+$secondary: #616070;
+$blue-accent: #00acdc;
```

Comment:
Don't use name of the color in variable names(like blue", "gray"...). Better to name it like `$button-color`...